en/xml/dbc-fo.xml

```diff
@@ -15,10 +15,8 @@
 -->
 
 <!--<?xml-model href="file:5.1/dbref.rnc" type="application/relax-ng-compact-syntax"?>-->
-<chapter xml:id="dbc.fo" version="5.1"
-  xmlns="http://docbook.org/ns/docbook"
-  xmlns:xi="http://www.w3.org/2001/XInclude"
-  xmlns:xlink="http://www.w3.org/1999/xlink">
+<chapter xml:id="dbc.fo" version="5.1" xmlns="http://docbook.org/ns/docbook"
```

Comment:
I guess, oXygen screwed up some lines. However, only the added `xi:include` line is relevant to this diff.

Could you revert this file and do it again so it only shows the respective line?